api/config/v1alpha1/envoygateway_types.go

```diff
@@ -15,6 +15,10 @@ const (
 	KindEnvoyGateway = "EnvoyGateway"
 	// GatewayControllerName is the name of the GatewayClass controller.
 	GatewayControllerName = "gateway.envoyproxy.io/gatewayclass-controller"
+	// GatewayAdminPort is the port which envoy gateway admin server is listening on.
+	GatewayAdminPort = 8899
```

Comment:
change this to 19000, same as dataplane?

Reply:
we discussed this in the community meeting today and majority agreed to reuse the same port number as the admin server used in envoy, which is `19000`, ( one less thing to remember :) )

Reply:
Done